packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -98,6 +99,34 @@ function ScreenRoot() {
 					</NavigationButtonAsItem>
 				</ItemGroup>
 			</CardBody>
+
+			<CardDivider />
+
+			<CardBody>
+				<Spacer
+					as="p"
+					paddingTop={ 2 }
+					paddingX="13px"
+					marginBottom={ 4 }
+				>
+					{ __(
+						'Add CSS beyond the styles offered by the existing design tools.'
```

Comment:
```suggestion
						'Add your own CSS to customize the appearance and layout of your site.'
```

Totally optional :)

Reply:
Yep, thanks, more meaningful to the average punter.